modules/migration-compatibility-guidelines.adoc

```diff
@@ -10,16 +10,43 @@
 
 You must install the {mtc-full} ({mtc-short}) Operator that is compatible with your {product-title} version.
 
-You cannot install {mtc-short} {mtc-version} on {product-title} 4.5, or earlier versions, because the custom resource definition API versions are incompatible.
+.Definitions
 
-You can migrate workloads from a source cluster with {mtc-short} {mtc-legacy-version-z} to a target cluster with {mtc-short} {mtc-version} as long as the `MigrationController` custom resource and the {mtc-short} web console are running on the target cluster.
+legacy platform:: {product-title} 4.5 and earlier
+modern platform:: {product-title} 4.6 and later
+legacy operator:: The {mtc-short} Operator installed on a legacy platform
+modern operator:: The {mtc-short} Operator installed
```

Comment:
This isn't complete, I'd also suggest: designed for modern platforms.

Reply:
Fixed.